cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,34 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.1.1 <http://docs.cask.co/cdap/3.1.1/index.html>`__
+=============================================================
+
+Bug Fixes
+---------
+
+- `CDAP-3259 <https://issues.cask.co/browse/CDAP-3259>`__ -
+  Removed a "live re-load" script accidentally included in the previous release.
```

Comment:
Is there a better way to phrase this :) In the sense - why wasn't it supposed to be in 3.1? The way it is currently, it doesn't look convincing.